nucypher/datastore/datastore.py

```diff
@@ -183,7 +183,8 @@ def attach_kfrag_to_saved_arrangement(self, alice, id_as_hex, kfrag, session=Non
         if policy_arrangement.alice_verifying_key.key_data != alice.stamp:
             raise alice.SuspiciousActivity
 
-        policy_arrangement.kfrag = bytes(kfrag)
+        # TODO:  is this ok?
+        session.query(PolicyArrangement).filter_by(id=id_as_hex.encode()).update({"kfrag": bytes(kfrag)})
```

Comment:
Despite this change, the fragment is still not being persisted.  Are we missing a call to session.add ?

Reply:
Let's remove these part of the chagseset for this PR to focus on the other the API change, leaving the DB handling for follow-up WRT #2088